src/main/groovy/net/nemerosa/versioning/git/GitInfoService.groovy

```diff
@@ -21,9 +21,9 @@ class GitInfoService implements SCMInfoService {
     SCMInfo getInfo(Project project, VersioningExtension extension) {
         // Is Git enabled?
         boolean hasGit = project.rootProject.file('.git').exists() ||
-                project.file('.git').exists() ||
-                (extension.gitRepoRootDir != null &&
-                        new File(extension.gitRepoRootDir, '.git').exists())
+        project.file('.git').exists() ||
+        (extension.gitRepoRootDir != null &&
+        new File(extension.gitRepoRootDir, '.git').exists())
```

Comment:
Cosmetic, but can you restore the initial indentation? I don't think this code has changed.